etc/tempesta_fw.conf

```diff
@@ -698,9 +698,9 @@
 #   Directive proxy_pass is mandatory for "vhost" and "location" directive
 #   blocks. For explicit default vhost directive (and included locations)
 #   only proxy_pass directives with 'default' server groups is allowed to be
-#   specified (or proxy_pass directive may be absent at all). For implicit
-#   default vhost - proxy_pass directive is not allowed (since default vhost
-#   must point to default server group).
+#   specified (or proxy_pass directive may be absent at all. In this case it
+#   is created automatically). For implicit default vhost - proxy_pass directive
+#   is not allowed (since default vhost must point to default server group).
```

Comment:
'For implicit default vhost - proxy_pass directive is not allowed (since default vhost must point to default server group).' -> 'For the implicit "defalt" vhost only `default` value for `proxy_pass` directive is allowed. It's created automatically if "default" server group is defined, explicit `proxy_pass` directive is not allowed in this case.'